### PR TITLE
CORE-12165: undo revert of revert and remove limit and time to live on key rotation

### PR DESCRIPTION
**Pull request by dickon**
Also add  key type on individual key rotation messages, and clarify docstrings.

### CODE REVIEW

File: data/avro-schema/src/main/resources/avro/net/corda/data/crypto/wire/ops/key/rotation/IndividualKeyRotationResponse.avsc
Comment (anien) [inline data/avro-schema/src/main/resources/avro/net/corda/data/crypto/wire/ops/key/rotation/IndividualKeyRotationResponse.avsc:1]: Shouldn't we also change `oldKeyAlias` and `newKeyAlias` to `oldKeyParentAlias` and `newKeyParentAlias` to keep consistent with the `IndividualKeyRotationRequest`?

Comment (dickon) [reply data/avro-schema/src/main/resources/avro/net/corda/data/crypto/wire/ops/key/rotation/IndividualKeyRotationResponse.avsc:1]: yes, done

File: data/avro-schema/src/main/resources/avro/net/corda/data/crypto/wire/ops/key/rotation/IndividualKeyRotationResponse.avsc [hunk 1]
```diff
@@ -27,7 +27,7 @@
     {
       "name": "targetKeyAlias",
       "type": "string",
-      "doc": "Specifies the wrapped key to rotate"
+      "doc": "Specifies the alias of the key that should be rotated"
```
Comment (anien) [inline data/avro-schema/src/main/resources/avro/net/corda/data/crypto/wire/ops/key/rotation/IndividualKeyRotationResponse.avsc]: Let's add dot at the end of the sentence.

Comment (dickon) [reply data/avro-schema/src/main/resources/avro/net/corda/data/crypto/wire/ops/key/rotation/IndividualKeyRotationResponse.avsc]: done

**Review by anien — APPROVED** [review]

**Pull request merged**